src/main/java/com/dreammaster/gthandler/recipes/FluidSolidifierRecipes.java

```diff
@@ -27,8 +28,12 @@
 
 public class FluidSolidifierRecipes implements Runnable {
 
+    ItemStack missing = new ItemStack(Blocks.fire);
```

Comment:
why?

Reply:
As example i used CompressorRecipes.java. I repeated how it was done there.

Reply:
1) you are not using it 2) never use this for gt machine recipes, it just hides broken stuff

Reply:
fair.